DynamicTablesPkg/Include/AcpiObjects.h

```diff
@@ -139,6 +140,107 @@ typedef struct AmlPsdInfo {
   UINT32    NumProc;
 } AML_PSD_INFO;
 
+/** A structure that describes a
+    processor power state control package Info.
+
+  Cf. ACPI 6.4, s8.4.2.1 _CST (Processor Power State Control).
+  Package {
+    Register  // Buffer (Resource Descriptor)
+    Type      // Integer (BYTE)
+    Latency   // Integer (WORD)
+    Power     // Integer (DWORD)
+  }
+*/
+typedef struct AmlCstPkgInfo {
+  EFI_ACPI_6_4_GENERIC_ADDRESS_STRUCTURE    Register;
+  UINT8                                     Type;
+  UINT16                                    Latency;
+  UINT32                                    Power;

```

Comment:
Is it possible to add comments in the structure like the other structures already present ?

```
/// Average power consumption of the C-State in mW
UINT32  Power;
```

Same comment for other fields and other new structures